spec/features/preassembly_run/hierarchical_files_spec.rb

```diff
@@ -33,50 +33,85 @@
     end
   end
 
-  it 'runs successfully and creates log file' do
```

Comment:
same comment as above, context blocks for failed and success based on file structure type selected